tasks/deploy/config/sepolia/ezeth-14day.ts

```diff
@@ -1,59 +1,71 @@
 import { parseEther } from "viem";
-import { HyperdriveInstanceDeployConfigInput } from "../../lib";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
 
-const CONTRIBUTION = "500";
+const CONTRIBUTION = parseEther("500");
 
-export const SEPOLIA_EZETH_14DAY: HyperdriveInstanceDeployConfigInput = {
+export const SEPOLIA_EZETH_14DAY: HyperdriveInstanceConfig<"EzETH"> = {
     name: "EZETH_14_DAY",
-    contract: "EzETHHyperdrive",
-    coordinatorName: "EZETH_COORDINATOR",
-    deploymentId: "0x666",
-    salt: "0xababe",
+    prefix: "EzETH",
+    coordinatorAddress: async (hre) =>
+        hre.hyperdriveDeploy.deployments.byName("EZETH_COORDINATOR").address,
+    deploymentId: toBytes32("EZETH_14_DAY"),
+    salt: toBytes32("0xababe"),
+    extraData: "0x",
     contribution: CONTRIBUTION,
-    fixedAPR: "0.05",
-    timestretchAPR: "0.05",
+    fixedAPR: parseEther("0.05"),
+    timestretchAPR: parseEther("0.05"),
+    targetCount: 4,
     options: {
-        // destination: "0xsomeone", defaults to deployer
+        destination: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
         asBase: false,
-        // extraData: "0x",
+        extraData: "0x",
     },
-    poolDeployConfig: {
-        baseToken: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
-        vaultSharesToken: {
-            name: "EZETH",
-            deploy: async (hre) => {
-                let vaultSharesToken = await hre.viem.getContractAt(
-                    "MockEzEthPool",
-                    hre.hyperdriveDeploy.deployments.byName("EZETH").address,
-                );
-                let pc = await hre.viem.getPublicClient();
-                // mint the contribution
-                let tx = await vaultSharesToken.write.mint([
-                    parseEther(CONTRIBUTION),
-                ]);
-                // approve the coordinator
-                tx = await vaultSharesToken.write.approve([
-                    hre.hyperdriveDeploy.deployments.byName("EZETH_COORDINATOR")
-                        .address,
-                    parseEther(CONTRIBUTION) + parseEther("10"),
-                ]);
-                await pc.waitForTransactionReceipt({ hash: tx });
+    prepare: async (hre) => {
+        let vaultSharesToken = await hre.viem.getContractAt(
+            "MockEzEthPool",
+            hre.hyperdriveDeploy.deployments.byName("EZETH").address,
+        );
+        let pc = await hre.viem.getPublicClient();
+        // mint the contribution
+        let tx = await vaultSharesToken.write.mint([CONTRIBUTION]);
+        // approve the coordinator
+        tx = await vaultSharesToken.write.approve([
+            hre.hyperdriveDeploy.deployments.byName("EZETH_COORDINATOR")
+                .address,
+            CONTRIBUTION + parseEther("10"),
```

Comment:
Why are we approving with this amount?

Reply:
For some reason (unsure why) we run into issues with insufficient approval amount on this contract. Rounding error maybe? IIRC you and I both encountered it when we ran the deploy script over vc last week.

Will reduce it to `0.01` ether since that should be sufficient